### ISSUE TITLE
Support T450S?

### ISSUE THREAD

**Issue by longpro1601** [jsassu20/Lenovo-ThinkPad-T450-Mojave#13]
Could I use this config for my T450S?

**Comment by svobluk** [comment]
shouldn't be problem, I guess specs will be quite a same.

**Comment by jsassu20** [comment]
Yes, T450s is Broadwell based. The only hardware differences between the T450 and T450s are unrelated to anything which would cause this configuration to require different settings. The T450s has a smaller form factor, and the T450s has one ram chip soldered to the mother board and one normal ram slot which can be upgraded, whereas the T450 has 2 changeable ram slots. Also I'm not sure if the T450s can support a smart card or not. Those are the only differences. All of the most important hardware is identical in both machines.

![Screen Shot 2019-06-26 at 5 04 02 PM](https://user-images.githubusercontent.com/12767985/60215404-898e3e00-9835-11e9-92d8-0a4a6eb41bd3.png)

![Screen Shot 2019-06-26 at 5 07 13 PM](https://user-images.githubusercontent.com/12767985/60215416-8eeb8880-9835-11e9-9c1b-615bc3981598.png)

**Comment by jianzhao0806** [comment]
I tried 10.14.3 , when I try to drive the audio, the Applealc.kext seems cause the reboot ,when I use -v.

**Comment by jianzhao0806** [comment]
> Yes, T450s is Broadwell based. The only hardware differences between the T450 and T450s are unrelated to anything which would cause this configuration to require different settings. The T450s has a smaller form factor, it has one 4GB ram chip soldered to the motherboard and one normal ram slot which can be upgraded, whereas the T450 has 2 changeable ram slots (but it also supports 20GB max vs 16GB in the T450), and the the T450s has a larger processor cache at 4mb vs 3mb in the T450. Also I'm not sure if the T450s can support a smart card or not. Those are the only differences. All of the most important hardware is identical in both machines.
> 
> The Intel CPU portion for both machines is a 5th generation i5 or i7 depending on which configuration you went with when you bought it but it irrelevant since they are both Broadwell generation processors.
> 
> ![Screen Shot 2019-06-26 at 5 04 02 PM](https://user-images.githubusercontent.com/12767985/60216052-0bcb3200-9837-11e9-9272-51a5e0656a03.png)
> 
> ![Screen Shot 2019-06-26 at 5 07 13 PM](https://user-images.githubusercontent.com/12767985/60215662-1a651980-9836-11e9-9564-f8f6b8f5cf70.png)

hi, I found a new problem. when I try to wake up my t450s. It is blackscreen and there have several buzzers. I can just force reboot.

**Comment by Bitbanger1900** [comment]
I run 10.15.4 with this configuration on a T450S. 

I Only needed to change the SystemProductName to MacBookPro12,1 in the config.plist otherwise it "hangs" during the boot process.
